packages/contracts-bedrock/src/L1/OPContractsManager.sol

```diff
@@ -445,7 +450,6 @@ contract OPContractsManager is ISemver, Initializable {
 
     /// @notice Helper method for encoding the SystemConfig initializer data.
     function encodeSystemConfigInitializer(
-        bytes4 _selector,
```

Comment:
And similarly why were we able to delete this?

Reply:
The `initialize` selector no longer gets passed through to opcm via the `Implementation` [struct](https://github.com/ethereum-optimism/optimism/blob/50564d430460a4dc7b63787814bd322751660f4a/packages/contracts-bedrock/src/L1/OPContractsManager.sol#L95). We know that the initializer for this SystemConfig will be the non-interop SystemConfig `initialize` selector because OPContractsManagerInterop overrides `encodeSystemConfigInitializer` to provide the SystemConfigInterop `initialize` function. 

I'm following the same logic for the rest of the contracts. e.g. [here](https://github.com/ethereum-optimism/optimism/pull/12851/files#diff-32c676c7a2cc25dc934a436546eccc2c3484b44491cc1569dce382ffb13b84c7R285) for L1ERC721Bridge initializer selector.
```solidity
data = encodeL1ERC721BridgeInitializer(IL1ERC721Bridge.initialize.selector, output);
```

If you think it should still be a function param I can make that change.